### PR TITLE
Updated azure-cosmos, spring-data-cosmos, and azure-cosmos-encryption release versions

### PR DESCRIPTION
**Pull request by kushagraThapar**
#### Key Bugs Fixes
* Fixed an issue in `CosmosPagedIterable` resulting in excessive memory consumption due to unbounded prefetch of pages when converting the `CosmosPagedIterable` into an `Iterator<FeedResponse<T>>`. - See [PR 27237](https://github.com/Azure/azure-sdk-for-java/pull/27237) and [PR 27299](https://github.com/Azure/azure-sdk-for-java/pull/27299)
* Fixed a `NullPointerException` in `CosmosDiagnostics isDiagnosticsCapturedInPagedFlux` - See [PR 27261](https://github.com/Azure/azure-sdk-for-java/pull/27261)
* Fixed an issue with allowing null values for add, set and replace operations in Patch API.
* Fixed an issue with top query when top x is greater than the total number of items in the database.
* Fixed synchronized lists and maps for order by query race condition.

#### Features Added
* Updated `Spring Boot` to version `2.6.3`.
* Updated `azure-cosmos` to version `4.27.0`.
* Add support for container `UniqueKey` policies.

#### New Features
* Added `Patch API` for encryption.
* Added `Bulk API` for encryption.
* Added `CosmosEncryptionClientBuilder` for creating `CosmosEncryptionAsyncClient` and `CosmosEncryptionClient`.
* Renamed source package to `com.azure.cosmos.encryption` inline with artifact id.
* Updated `azure-cosmos` to version `4.27.0`.

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-#### Breaking Changes
-
-#### Bugs Fixed
+### 4.27.0 (2022-03-10)
+#### Key Bugs Fixes
 * Fixed an issue in `CosmosPagedIterable` resulting in excessive memory consumption due to unbounded prefetch of pages when converting the `CosmosPagedIterable` into an `Iterator<FeedResponse<T>>`. - See [PR 27237](https://github.com/Azure/azure-sdk-for-java/pull/27237) and [PR 27299](https://github.com/Azure/azure-sdk-for-java/pull/27299)
 * Fixed a `NullPointerException` in `CosmosDiagnostics isDiagnosticsCapturedInPagedFlux` - See [PR 27261](https://github.com/Azure/azure-sdk-for-java/pull/27261)
-
-#### Other Changes
+* Fixed an issue with allowing null values for add, set and replace operations in Patch API.
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:7]: Q: should we be consistently add the link to PR?

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ## Release History
 
-### 3.19.0-beta.1 (Unreleased)
-
+### 3.19.0 (2022-03-10)
 #### Features Added
+* Updated `Spring Boot` to version `2.6.3`.
+* Updated `azure-cosmos` to version `4.27.0`.
+* Add support for container `UniqueKey` policies.
```
Comment (xinlian12) [inline sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:7]: Add -> Added

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:7]: Good point, yes, we should do that, I will update it, thanks!

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:7]: Got it, updated.

**Review by vcolin7 — APPROVED** [review]
LGTM

**Pull request merged**